### PR TITLE
Add documentation on supported entities

### PR DESCRIPTION
**Pull request by chester-leung**
Add documentation on the entity types (e.g. name, phone number, SSN) that PromptGuard supports sanitizing.

### CODE REVIEW

**Review by ggroode — COMMENTED** [review]
Why the three extra columns and rows?

File: docs/getting_started/overview.md [hunk 1]
```diff
@@ -1 +1,29 @@
-# Technical overview
\ No newline at end of file
+# Overview
+
+## Supported Entities
+
+Currently, the following entity types will be identified and sanitized:
+
+| Type                                                                                                                            | Notes                               |   |   |   |
```
Comment (zizhong) [inline docs/getting_started/overview.md]: Preview shows some empty rows and cols

**Comment by chester-leung** [comment]
@ggroode @zizhong thanks for catching this, I've cleaned it up

**Review by ggroode — APPROVED** [review]
Thanks, LGTM modulo CI passing.

File: docs/getting_started/overview.md [hunk 2]
```diff
@@ -1 +1,24 @@
...
\ No newline at end of file
+# Overview
+
+## Supported Entities
+
+Currently, the following entity types will be identified and sanitized:
+
+| **Type**                                                                                                                        | **Notes**                           |
+|---------------------------------------------------------------------------------------------------------------------------------|-------------------------------------|
+| Bank account numbers                                                                                                            |                                     |
```
Comment (zizhong) [inline docs/getting_started/overview.md:9]: sort the items?

**Review by zizhong — APPROVED** [review]
LG

File: docs/getting_started/overview.md (hunk 2 above)
Comment (chester-leung) [reply docs/getting_started/overview.md:9]: Done

**Pull request merged**